miru/ext/nav/navigator.py

```diff
@@ -45,7 +45,7 @@ def __init__(
         timeout: t.Optional[t.Union[float, int, datetime.timedelta]] = 120.0,
         autodefer: bool = True,
     ) -> None:
-        self._pages: t.Sequence[t.Union[str, hikari.Embed]] = pages
+        self._pages: t.Sequence[t.Union[str, hikari.Embed, t.Sequence[hikari.Embed]]] = pages
```

Comment:
Don't forget to update the parameter type hint and doc string :)

Reply:
Apologies, I believe everything should be properly hinted now!